ml/build.gradle

```diff
@@ -3,14 +3,6 @@ description = "Sparkling Water ML Pipelines"
 apply from: "$rootDir/gradle/sparkTest.gradle"
 apply from: "$rootDir/gradle/utils.gradle"
 
-sourceSets {
-  main {
```

Comment:
Actually, we don't need distinction on spark-version basis in ml package :) Thanks @mn-mikke !!